force-app/main/default/layouts/Case-SA Request.layout-meta.xml

```diff
@@ -126,10 +122,6 @@
                 <behavior>Edit</behavior>
                 <field>Provider_Validated__c</field>
             </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>Provider_Validated_Date__c</field>
-            </layoutItems>
```

Comment:
@marklise did you intend to remove `Provider_Validated_Date__c` and `Patient_Validated_Date__c` from the layout?

Reply:
No - I'll update that

Reply:
Fixed in 223b066.